docs/content/preview/benchmark/resilience/_index.md

```diff
@@ -12,20 +12,29 @@ menu:
     weight: 21
 type: indexpage
 ---
-For a tutorial that explores how YugabyteDB handles component failure and provides high availability (HA), see [Fault tolerance](../../../explore/fault-tolerance).
-
-<div class="row">
-
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-      <a class="section-link icon-offset" href="jepsen-testing-ysql/">
-          <div class="head">
-              <img class="icon" src="/images/section_icons/explore/high_performance.png" aria-hidden="true" />
-              <div class="title">Jepsen testing</div>
-          </div>
-          <div class="body">
-              Read the Jepsen test results for YSQL and YCQL.
-          </div>
-      </a>
-  </div>
-
-</div>
+
+Resiliency refers to the ability of a system to withstand and recover from failures or disruptions, whether they are caused by software bugs, hardware issues, network problems, or external events. A resilient system is designed to absorb the impact of failures and continue operating, even if at a degraded level, without experiencing a complete outage.
+
+In YugabyteDB, resiliency is achieved through various techniques. Some of them are,
+
+- **Fault tolerance** : Replicating tablets on to multiple nodes with one acting as a leader and others as followers. If the leader fails, a new leader is automatically elected, ensuring continuous availability. This replication and fault-tolerant architecture allows the database to withstand the failure of individual nodes or even entire data centers without losing data or service availability.
+- **Consistency Guarantees** : Raft based consensus ensures full ACID (Atomicity, Consistency, Isolation, Durability) transactions, even across multiple tablets and data centers. This consistency model helps maintain data integrity and coherence, even in the face of failures or network partitions.
+- **Self-Healing** : YugabyteDB automatically detects and recovers from failures, such as node crashes, disk failures, or network partitions. It can automatically repair and rebalance the cluster by re-replicating data and redistributing tablet leaders to maintain optimal performance and resilience.
```

Comment:
```suggestion
- **Self-healing** : YugabyteDB automatically detects and recovers from failures, such as node crashes, disk failures, or network partitions. It can automatically repair and rebalance the cluster by re-replicating data and redistributing tablet leaders to maintain optimal performance and resilience.
```